cli/cli.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import sys
+
+from configargparse import ArgParser
+
+from cli.commands.create_submission_pipeline_command import (
+    CreateSubmissionPipelineCommand,
+)
+from cli.commands.create_benchmark_pipeline_command import (
+    CreateBenchmarkPipelineCommand,
+)
+
+
+class Cli:
+    def __init__(self):
+        self.__commands = {
+            "create-benchmark-pipeline": CreateBenchmarkPipelineCommand(),

```

Comment:
This is the correct kebab-case here, your example in the README is incorrect snake case.